dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/ContainerHolder.java

```diff
@@ -0,0 +1,47 @@
+package com.dotmarketing.portlets.templates.design.bean;
+
+import com.dotmarketing.portlets.templates.design.util.PreviewTemplateUtil;
+
+import java.util.List;
+
+/**
+ * Created by freddyrodriguez on 11/13/17.
+ */
+public class ContainerHolder {
+
+    private boolean preview;
+    private List<String> containers;
+
+    public List<String> getContainers() {
+        return containers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "containers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)